config/formats.ts

```diff
@@ -199,7 +196,7 @@ const Formats = [
 		],
 
 		mod: 'gen8',
-		ruleset: ['[Gen 8] OU', '-Melmetal', '+CAP'],
+		ruleset: ['[Gen 8] OU', '+CAP'],
```

Comment:
This seems like a weirdly consequential change to do in a TS refactor. Can it (and similar changes) be pulled out into a separate PR?

Reply:
I mean, I guess? They are remnants of the OU Melmetal suspect test that happened ages ago, so it seems very unproblematic to do this here.